google/auth/aws.py

```diff
@@ -349,35 +348,15 @@ class Credentials(external_account.Credentials):
     def __init__(
         self,
         audience,
-        subject_token_type,
-        token_url,
         credential_source=None,
-        service_account_impersonation_url=None,
-        service_account_impersonation_options={},
-        client_id=None,
-        client_secret=None,
-        quota_project_id=None,
-        scopes=None,
-        default_scopes=None,
+        **kwargs,
     ):
         """Instantiates an AWS workload external account credentials object.
 
         Args:
-            audience (str): The STS audience field.
```

Comment:
This class should still take the same args as before, right?

Reply:
The args are still the same as before.  Should I list out all of the parameters here, or should I mention kwargs, and to look at external_account.Credentials to see all of the values that can be set there?

Reply:
I guess it's a balance between user convenience and maintainability.  It looks like we haven't done a 100% job at maintaining these as new arguments have gotten added, but I'll fix them up for now

Reply:
Added documentation for args and kwargs